supported_os_data.yml

```diff
@@ -1,4 +1,4 @@
-current_os: 12.0
+current_os: 12.3
```

Comment:
We're not updating this just yet, otherwise we'd be updating it with ever release of macOS. Should probably do the same with the Big Sur releases on lines 115, 122, 130, and 139 as well. At the time we didn't know exactly how Apple was going to be doing updates versions for N-1/-2. Might also need to add the original OS shipping versions for 2021 Macs.

Reply:
I personally don't mind updating it every couple months when the new OS patches come out. (But then again I'm not the one merging the PRs.)

I'll dig up the details for the other new models that aren't on the list yet and add them as well.